files/settings-default.cfg

```diff
@@ -350,6 +350,10 @@ terrain specular map pattern = _diffusespec
 # Affected objects use shaders.
 apply lighting to environment maps = false
 
+# Use Euclidean distance to determince fog intensity instead of plane.
+# Affected objects use shaders.
```

Comment:
Determine fog intensity based on the distance from the eye point. <newline>
This makes fogging independent from the viewing angle. Shaders will be used to render all objects.